src/Pulp/System/FFI.purs

```diff
@@ -21,6 +21,8 @@ foreign import data NodeError :: *
 
 type PulpEffects = (node :: Node, console :: CONSOLE, buffer :: BUFFER, fs :: FS, avar :: AVAR, err :: EXCEPTION, process :: PROCESS, cp :: CHILD_PROCESS, http :: HTTP, readline :: READLINE)
 
+type PulpEffectsSansErr = (node :: Node, console :: CONSOLE, buffer :: BUFFER, fs :: FS, avar :: AVAR, process :: PROCESS, cp :: CHILD_PROCESS, http :: HTTP, readline :: READLINE)
```

Comment:
I was going to use it in Pulp.Watch since the type of launchAff changed; I ultimately just used the old version of launchAff.  Going to revert this line.